tutorials/hsdk-ngc-container/.vscode/launch.json

```diff
@@ -0,0 +1,113 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    // https://code.visualstudio.com/docs/editor/variables-reference
+    "configurations": [
+        {
+            "name": "Python: Debug Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "autoReload": {
+                "enable": true
+            },
+            "cwd": "/workspace/build",
+            "env": {
+                "PYTHONPATH": "/opt/nvidia/holoscan/python/lib",
+                "HOLOSCAN_LOG_LEVEL": "DEBUG",
+                "HOLOSCAN_EXECUTOR_LOG_LEVEL": "INFO",
+                "HOLOSCAN_TESTS_DATA_PATH": "/opt/nvidia/holoscan/tests/data",
+                "HOLOSCAN_INPUT_PATH": "/opt/nvidia/holoscan/data",
```

Comment:
I guess if we properly set this environment variable in devcontainer.json, this line may not be needed.

Reply:
I've removed all files inside `.vscode/` directory since we are copying it from the NGC container.